test/test_buffer.cpp

```diff
@@ -138,7 +155,7 @@ namespace sparrow
         TEST_CASE("front")
         {
             const std::size_t size = 4u;
-            const std::uint8_t expected_value = 3u;
+            const std::int32_t expected_value = 3u;
```

Comment:
Note that the `u` is not necessary anymore here (same with other values below)

Reply:
Ok done